kubernetes/linux/main.sh

```diff
@@ -869,11 +871,11 @@ else
       mdsd ${MDSD_AAD_MSI_AUTH_ARGS} -r ${MDSD_ROLE_PREFIX} -e ${MDSD_LOG}/mdsd.err -w ${MDSD_LOG}/mdsd.warn -o ${MDSD_LOG}/mdsd.info -q ${MDSD_LOG}/mdsd.qos 2>>/dev/null &
 fi
 
-# Set up a cron job for logrotation
-if [ ! -f /etc/cron.d/ci-agent ]; then
-      echo "setting up cronjob for ci agent log rotation"
-      echo "*/5 * * * * root /usr/sbin/logrotate -s /var/lib/logrotate/ci-agent-status /etc/logrotate.d/ci-agent >/dev/null 2>&1" >/etc/cron.d/ci-agent
-fi
+# # Set up a cron job for logrotation
```

Comment:
if this commented, how does the log rotation work?

Reply:
Ohh I was using crontab. But figured that this works as well